.github/workflows/r_docs.yml

```diff
@@ -0,0 +1,46 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+name: R documentation
+
+on:
+  push:
+    paths:
+      - '.github/workflows/r_docs.yml'
+      - 'r/R/*.R'
+
+jobs:
+  roxygen:
+    name: Build and Push
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
```

Comment:
We can store credential with `actions/checkout@v2`: https://github.com/actions/checkout/tree/v2

It means that we can use `git push origin ...` instead of `git push https://${{github.actor}}:${{secrets.GITHUB_TOKEN}}@github.com/...`.